### ISSUE TITLE
Clean Up Index Layout

### ISSUE THREAD

**Issue by philip-ulrich** [cydrobolt/polr#268]
## Expected Behavior
The spacing on the index should be more spaced out so that there is not as much "whitespace" and so that it doesn't look cramped. 

## Current Behavior
The current layout looks very claustrophobic. 

## Possible Solution
Move tips to the bottom of the page. Add more padding around things. A rough implementation of this is live at https://z3r0.tech. The follow is the changes:

* `...` indicates omitted unchanged section of file
index.blade.php
```
...
    <input type='submit' class='btn btn-info' id='shorten' value='Shorten' />
    <a href='#' class='btn btn-warning' id='show-link-options'>Link Options</a>
    <input type="hidden" name='_token' value='{{csrf_token()}}' />
</form>

@endsection

<div id='tips' class='text-muted tips'>
    <i class='fa fa-spinner'></i> Loading Tips...
</div>
@section('js')
<script src='js/index.js'></script>
@endsection
```

index.css
```
.custom-url-field {
    display: inline;
    width: 175px;
    font-size: 30px;
    margin: 20px 0;
}
...
.long-link-input {
    margin-bottom: 65px;
}
.tips {
    bottom: 20px;
    width: 100%;
    position: fixed;
    text-align: center;
}
...
#link-availability-status {
    margin-bottom: 20px;
}
...
h1 {
    padding: 50px;
}
```

## Context
It bothers my inner designer.

--------------------

Ultimately this could be considered preference and is really a minor enhancement, but something to consider. CSS could use some work. Currently causes bugs like issues with the tips section when page height exceeds height of the string.

**Comment by cydrobolt** [comment]
I agree. Ultimately, the index page is largely down to preference and it would be difficult to find a formula that fits all users. However, I have considered providing multiple index "templates" that could be used for different types of users. 

For example, enterprise users who do not have any user-facing components at all might like a more compact and informational index page rather than the spaced-out page that has been available and that you've presented. I feel like such a change might be more helpful to users, whereas spacing should definitely be tweakable by interested users.

Fundementally, Polr is free and open source, so there are no restrictions as to what kind of layout users can implement. It's up to them to select what works best.

**Comment by philip-ulrich** [comment]
Makes sense. I totally get that there are lots of different uses and preferences for how the page should look. Ultimately having different index "templates" might be a nice configurable option, but at the same time the people who want to change it will definitely make those changes themselves. I think the spaced-out page probably will hit what the "average joe" wants more than what there is currently.

This is just my 2 cents though. I don't want to open a can of worms here on which look is "better" and which isn't. More or less just trying to help contribute a little to this cool project. 👍

**Comment by overint** [comment]
I think so changes have been made since this issue, what are your thoughts on the current layout?
